File: spec/python/test_imports0.py
import unittest

from imports0 import Imports0

class TestImports0(unittest.TestCase):
    def test_imports0(self):
        r = Imports0.from_file("src/fixed_struct.bin")

        self.assertEqual(r.two, 0x50)
        self.assertEqual(r.hw.one, 0x41)
        self.assertEqual(r.hw_one, 0x41)

File: spec/lua/test_imports0.lua
local luaunit = require("luaunit")

require("imports0")

TestImports0 = {}

function TestImports0:test_imports0()
    local r = Imports0:from_file("src/fixed_struct.bin")

    luaunit.assertEquals(r.two, 0x50)
    luaunit.assertEquals(r.hw.one, 0x41)
    luaunit.assertEquals(r.hw_one, 0x41)
end
